src/plots/cartesian/layout_attributes.js

```diff
@@ -513,6 +513,16 @@ module.exports = {
     tickformatstops: {
         _isLinkedToArray: 'tickformatstop',
 
+        visible: {
```

Comment:
Maybe `enabled` (similar to transforms) here would make more sense.

Reply:
.... but maybe it isn't worth breaking symmetric with all other templated arrays :thinking:

Reply:
Right, `enabled` certainly makes more sense here. I thought about that but deferred to symmetry at the time. But the way this ended up working out, it would be easy to make `visibilityAttr: 'enabled'` or something an option to `handleArrayContainerDefaults`. I'll give that a try.

Reply:
switched to `enabled` in https://github.com/plotly/plotly.js/pull/2761/commits/fb489aa0fcd2c526335f09d2068f3659683ad10d

Reply:
I'm glad this wasn't too hard to implement :tada: